src/main/java/uk/gov/hmcts/divorce/common/event/SubmitAos.java

```diff
@@ -111,10 +129,12 @@ private List<String> validateAos(final AcknowledgementOfService acknowledgementO
     private PageBuilder addEventConfig(ConfigBuilder<CaseData, State, UserRole> configBuilder) {
         return new PageBuilder(configBuilder
             .event(SUBMIT_AOS)
-            .forStateTransition(AosDrafted, Holding)
+            .forStates(ArrayUtils.add(AOS_STATES, AosDrafted))
```

Comment:
same here use `EnumSet.of`

Reply:
this can be done but not the others, EnumSet.of is an overloaded method and one of the arguments is single element and accepts the rest of the elements as array.